.gitlab-ci.yml

```diff
@@ -0,0 +1,14 @@
+image: node:8
+
+stages:
+  - test
```

Comment:
Should we also add a 'publish' stage here (like we do with eslint-config)?

Reply:
'deploy': https://github.com/Ultimaker/eslint-config/blob/master/.gitlab-ci.yml

Reply:
Yes, but only once there is something to deploy :)

Reply:
That's what the 'only' filter in the publish job is for. it only does the actual deploy when a tag is created.

Reply:
Added, can I get a 👍 so this can be merged?